src/workspace.rs

```diff
@@ -137,6 +137,28 @@ impl Manifest {
 
     /// Set `[profile.release]` lto flag
     pub fn with_profile_release_lto(&mut self, enabled: bool) -> Result<&mut Self> {
+        let lto = self
+            .get_profile_release_table_mut()?
+            .entry("lto")
+            .or_insert(enabled.into());
+        *lto = enabled.into();
+        Ok(self)
+    }
```

Comment:
Isn't this superseeded by `with_profile_release_defaults`?

Reply:
No because this will overwrite the user-defined `lto` value if present. This is required for `generate-metadata` when user has `[profile.release] lto = true`.